CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+:exclamation: = Breaking change
+
+## (unreleased)
+
+### Changed
+* :exclamation: Renamed the `clinlp_entity_matcher` to `clinlp_rule_based_entity_matcher`
+* :exclamation: `clinlp` now stores entities in `doc.spans['ents']` rather than `doc.ents`, allowing for overlap
```

Comment:
Contrast "new" conflict solution with previous behaviour (i.e. overlap vs. storing only the latest matched entity)

Reply:
Thanks, added some clarification here